cap_v4l.cpp of videoio make error

**System information (version)**

    OpenCV => 4.0.1
    Operating System / Platform => CentOS release 6.8
    Compiler => g++ (GCC) 4.8.5 (reinstalled gcc)

**Detailed description**

make error:

[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/videoio_registry.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/videoio_c.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/cap.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/cap_images.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/cap_mjpeg_encoder.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/cap_mjpeg_decoder.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/container_avi.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/cap_gstreamer.cpp.o
[ 45%] Building CXX object modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/cap_v4l.cpp.o
/home/lz/Downloads/opencv-4.0.1/modules/videoio/src/cap_v4l.cpp: In function ‘int cv::capPropertyToV4L2(int)’:
/home/lz/Downloads/opencv-4.0.1/modules/videoio/src/cap_v4l.cpp:1613:16: error: ‘V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_HEIGHT’ was not declared in this scope
         return V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_HEIGHT;
                ^
/home/lz/Downloads/opencv-4.0.1/modules/videoio/src/cap_v4l.cpp:1615:16: error: ‘V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_WIDTH’ was not declared in this scope
         return V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_WIDTH;
                ^
make[2]: *** [modules/world/CMakeFiles/opencv_world.dir/__/videoio/src/cap_v4l.cpp.o] Error 1
make[1]: *** [modules/world/CMakeFiles/opencv_world.dir/all] Error 2
make: *** [all] Error 2

--- a/modules/videoio/src/cap_v4l.cpp
+++ b/modules/videoio/src/cap_v4l.cpp
@@ -243,6 +243,14 @@ make & enjoy!
 #define V4L2_CID_ISO_SENSITIVITY (V4L2_CID_CAMERA_CLASS_BASE+23)
 #endif
 
+// https://github.com/opencv/opencv/issues/13929
+#ifndef V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_HEIGHT
+#define V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_HEIGHT (V4L2_CID_MPEG_BASE+364)
+#endif
+#ifndef V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_WIDTH
+#define V4L2_CID_MPEG_VIDEO_H264_VUI_EXT_SAR_WIDTH (V4L2_CID_MPEG_BASE+365)
+#endif
+
 /* Defaults - If your board can do better, set it here.  Set for the most common type inputs. */
 #define DEFAULT_V4L_WIDTH  640
 #define DEFAULT_V4L_HEIGHT 480

